### PR TITLE
MM-31203 UX Optimizations to make end users happier

### PR DESCRIPTION
**Pull request by cwarnermm**
Documentation for: https://mattermost.atlassian.net/browse/MM-31203

Updates:
- Admin Guide > Getting Started > Administrator Tasks: added new **User Experience Optimizations** section

### CODE REVIEW

File: source/deployment/on-boarding.rst [hunk 1]
```diff
@@ -67,3 +67,77 @@ Common Tasks
 **Upgrading Mattermost**
```
Comment (esethna) [inline source/deployment/on-boarding.rst:67]: This is now duplicated?

Comment (esethna) [reply source/deployment/on-boarding.rst:67]: Perhaps we just pull any unique info from here into below?

File: source/deployment/on-boarding.rst [hunk 2]
```diff
@@ -67,3 +67,77 @@ Common Tasks
...
+    - Create and share memes with `Memes <https://integrations.mattermost.com/?s=Memes%20Plugin>`__.
+    - Set personal reminders with `Remind <https://integrations.mattermost.com/?s=Remind%20Plugin>`__.
+    - Create and share to do items with `Todo <https://github.com/mattermost/mattermost-plugin-todo>`__.
+    - Customize welcome messages for new users with `WelcomeBot <https://integrations.mattermost.com/?s=WelcomeBot%20Plugin>`__.
+
+Explore all plugins and integrations available in the `Mattermost Plugin Marketplace <https://integrations.mattermost.com/>`__.
+
+**3. Enable Automatically Extended Sessions**
+
+Keep your desktop and mobile users logged in and refresh user sessions automatically by configuring the following: **System Console > Sessions > Extend session length with activity**: ``true``. See the `Extend session length with activity <https://docs.mattermost.com/administration/config-settings.html#extend-session-length-with-activity>`__ configuration settings documentation for details.
```
Comment (esethna) [inline source/deployment/on-boarding.rst:97]: Propose adding a link to blog for more details

File: source/deployment/on-boarding.rst [hunk 3]
```diff
@@ -67,3 +67,77 @@ Common Tasks
...
+
+Enable full content push notifications, including the sender’s name, the channel name, and the message text, by configuring the following: **System Console > Notifications > Push Notification Contents**: ``Full message contents``. See the `Push notification contents <https://docs.mattermost.com/administration/config-settings.html#push-notification-contents>`__ configuration settings documentation for details.
+
+.. note::
+
+  - Mattermost Enterprise E10 and E20 customers can `enable HPNS <https://docs.mattermost.com/mobile/mobile-hpns.html>`__ that includes production-level uptime SLAs.
+
+  - Mattermost Enterprise E20 customers can `enable ID-Only push notifications <https://docs.mattermost.com/administration/config-settings.html#push-notification-contents>`__ so the notification message contents are not passed through Apple Push Notification Service (APNS) or Google Firebase Cloud Messaging (FCM) before reaching the device.
```
Comment (esethna) [inline source/deployment/on-boarding.rst:109]: Include link to blog post for more details on ID only notifications

**Review by esethna — CHANGES_REQUESTED** [review]

File: source/deployment/on-boarding.rst (hunk 2 above)
Comment (justinegeffen) [inline source/deployment/on-boarding.rst:97]:
```suggestion
Keep your desktop and mobile users logged in and refresh user sessions automatically by setting **System Console > Sessions > Extend session length with activity** to ``true``. See the `Extend session length with activity <https://docs.mattermost.com/administration/config-settings.html#extend-session-length-with-activity>`__ configuration settings documentation for details.
```

File: source/deployment/on-boarding.rst [hunk 4]
```diff
@@ -67,3 +67,77 @@ Common Tasks
...
+
+Explore all plugins and integrations available in the `Mattermost Plugin Marketplace <https://integrations.mattermost.com/>`__.
+
+**3. Enable Automatically Extended Sessions**
+
+Keep your desktop and mobile users logged in and refresh user sessions automatically by configuring the following: **System Console > Sessions > Extend session length with activity**: ``true``. See the `Extend session length with activity <https://docs.mattermost.com/administration/config-settings.html#extend-session-length-with-activity>`__ configuration settings documentation for details. 
+
+**4. Enable Full Content Push Notifications**
+
+Enable push notifications on mobile devices to deliver messages in real time by configuring the following: **System Console > Push Notification Server > Enable Push Notifications**: ``Use TPNS``. See the `Push notification server <https://docs.mattermost.com/administration/config-settings.html#push-notification-server>`__ configuration settings documentation for details.
```
Comment (justinegeffen) [inline source/deployment/on-boarding.rst:101]:
```suggestion
Enable push notifications on mobile devices to deliver messages in real time by setting **System Console > Push Notification Server > Enable Push Notifications** to ``Use TPNS``. See the `Push notification server <https://docs.mattermost.com/administration/config-settings.html#push-notification-server>`__ configuration settings documentation for details.
```

File: source/deployment/on-boarding.rst [hunk 5]
```diff
@@ -67,3 +67,77 @@ Common Tasks
...
+
+**4. Enable Full Content Push Notifications**
+
+Enable push notifications on mobile devices to deliver messages in real time by configuring the following: **System Console > Push Notification Server > Enable Push Notifications**: ``Use TPNS``. See the `Push notification server <https://docs.mattermost.com/administration/config-settings.html#push-notification-server>`__ configuration settings documentation for details. 
+
+Enable full content push notifications, including the sender’s name, the channel name, and the message text, by configuring the following: **System Console > Notifications > Push Notification Contents**: ``Full message contents``. See the `Push notification contents <https://docs.mattermost.com/administration/config-settings.html#push-notification-contents>`__ configuration settings documentation for details.
+
+.. note::
+
+  - Mattermost Enterprise E10 and E20 customers can `enable HPNS <https://docs.mattermost.com/mobile/mobile-hpns.html>`__ that includes production-level uptime SLAs.
```
Comment (justinegeffen) [inline source/deployment/on-boarding.rst:107]:
```suggestion
  - Mattermost Enterprise Edition E10 and E20 customers can `enable HPNS <https://docs.mattermost.com/mobile/mobile-hpns.html>`__ that includes production-level uptime SLAs.
```

File: source/deployment/on-boarding.rst (hunk 3 above)
Comment (justinegeffen) [inline source/deployment/on-boarding.rst:109]:
```suggestion
  - Mattermost Enterprise Edition E20 customers can `enable ID-Only push notifications <https://docs.mattermost.com/administration/config-settings.html#push-notification-contents>`__ so the notification message contents are not passed through Apple Push Notification Service (APNS) or Google Firebase Cloud Messaging (FCM) before reaching the device.
```

File: source/deployment/on-boarding.rst [hunk 6]
```diff
@@ -67,3 +67,77 @@ Common Tasks
...
+
+  - Mattermost Enterprise E20 customers can `enable ID-Only push notifications <https://docs.mattermost.com/administration/config-settings.html#push-notification-contents>`__ so the notification message contents are not passed through Apple Push Notification Service (APNS) or Google Firebase Cloud Messaging (FCM) before reaching the device.
+
+**5. Enable Custom Emoji**
+
+`Emojis <https://docs.mattermost.com/help/messaging/emoji.html>`__ enable users to express concepts such as emotions and physical gestures in messages. Enable the emoji picker by configuring the following: **System Console > Emoji > Enable Emoji Picker**: ``true``. See the `Enable emoji picker <https://docs.mattermost.com/administration/config-settings.html#enable-emoji-picker>`__ configuration settings documentation for details.
```
Comment (justinegeffen) [inline source/deployment/on-boarding.rst:113]:
```suggestion
`Emojis <https://docs.mattermost.com/help/messaging/emoji.html>`__ enable users to express concepts such as emotions and physical gestures in messages. Enable the emoji picker by setting **System Console > Emoji > Enable Emoji Picker** to ``true``. See the `Enable emoji picker <https://docs.mattermost.com/administration/config-settings.html#enable-emoji-picker>`__ configuration settings documentation for details.
```

File: source/deployment/on-boarding.rst [hunk 7]
```diff
@@ -67,3 +67,77 @@ Common Tasks
...
+
+Empower users to create and share their own custom emojis by configuring the following: **System Console > Emoji > Enable Custom Emoji**: ``true``. See the `Enable custom emoji <https://docs.mattermost.com/administration/config-settings.html#enable-custom-emoji>`__ configuration settings documentation for details.
```
Comment (justinegeffen) [inline source/deployment/on-boarding.rst:115]:
```suggestion
Empower users to create and share their own custom emojis by setting **System Console > Emoji > Enable Custom Emoji** to ``true``. See the `Enable custom emoji <https://docs.mattermost.com/administration/config-settings.html#enable-custom-emoji>`__ configuration settings documentation for details.
```

File: source/deployment/on-boarding.rst [hunk 8]
```diff
@@ -67,3 +67,77 @@ Common Tasks
...
+
+**6. Enable GIF Picker (Beta)**
+
+GIFs are animated images that can make messaging more fun and engaging. Enable users to access the Mattermost GIF picker from the message draft area by configuring the following: **System Console > GIF (Beta) > Enable GIF Picker**: ``true``. See the `Enable GIF picker <https://docs.mattermost.com/administration/config-settings.html#enable-gif-picker>`__ configuration settings documentation for details.
```
Comment (justinegeffen) [inline source/deployment/on-boarding.rst:119]:
```suggestion
GIFs are animated images that can make messaging more fun and engaging. Enable users to access the Mattermost GIF picker from the message draft area by setting **System Console > GIF (Beta) > Enable GIF Picker** to ``true``. See the `Enable GIF picker <https://docs.mattermost.com/administration/config-settings.html#enable-gif-picker>`__ configuration settings documentation for details.
```

File: source/deployment/on-boarding.rst [hunk 9]
```diff
@@ -67,3 +67,77 @@ Common Tasks
...
+
+**7. Enable Link Previews**
+
+Link previews provide a visual glimpse of relevant content for links shared in messages. Enable link previews by configuring the following: **System Console > Posts > Enable Link Previews**: ``true``. See the `Enable link previews <https://docs.mattermost.com/administration/config-settings.html#enable-link-previews>`__ configuration settings documentation for details.
```
Comment (justinegeffen) [inline source/deployment/on-boarding.rst:123]:
```suggestion
Link previews provide a visual glimpse of relevant content for links shared in messages. Enable link previews by setting **System Console > Posts > Enable Link Previews** to ``true``. See the `Enable link previews <https://docs.mattermost.com/administration/config-settings.html#enable-link-previews>`__ configuration settings documentation for details.
```

File: source/deployment/on-boarding.rst [hunk 10]
```diff
@@ -67,3 +67,77 @@ Common Tasks
...
+ 
+**8. Enable Batched Email Notifications**
+
+Email notifications can be batched together so users don’t get overwhelmed with too many emails.
+
+Enable email notifications first by configuring the following: **System Console > Notifications > Enable Email Notifications**: ``true``. See the `Enable email notifications <https://docs.mattermost.com/administration/config-settings.html#enable-email-notifications>`__ configuration settings documentation for details. Email notifications require configuring an `SMTP email server <https://docs.mattermost.com/administration/config-settings.html#smtp-server>`__.
```
Comment (justinegeffen) [inline source/deployment/on-boarding.rst:129]:
```suggestion
Enable email notifications first by setting **System Console > Notifications > Enable Email Notifications** to ``true``. See the `Enable email notifications <https://docs.mattermost.com/administration/config-settings.html#enable-email-notifications>`__ configuration settings documentation for details. Note that email notifications require an `SMTP email server <https://docs.mattermost.com/administration/config-settings.html#smtp-server>`__ to be configured.
```

File: source/deployment/on-boarding.rst [hunk 11]
```diff
@@ -67,3 +67,77 @@ Common Tasks
...
+
+Then, enable batched email notifications by configuring the following: **System Console > Notifications > Enable Email Batching**: ``true``. See the `Enable email batching <https://docs.mattermost.com/administration/config-settings.html#enable-email-batching>`__ configuration settings documentation for details.Email batching is not available if you are running your deployment in `High Availability <https://docs.mattermost.com/deployment/cluster.html>`__.
```
Comment (justinegeffen) [inline source/deployment/on-boarding.rst:131]:
```suggestion
Then, enable batched email notifications by setting **System Console > Notifications > Enable Email Batching** to ``true``. See the `Enable email batching <https://docs.mattermost.com/administration/config-settings.html#enable-email-batching>`__ configuration settings documentation for details. Note that email batching is not available if you are running your deployment in `High Availability <https://docs.mattermost.com/deployment/cluster.html>`__.
```

File: source/deployment/on-boarding.rst [hunk 12]
```diff
@@ -67,3 +67,77 @@ Common Tasks
...
+
+**9. Enable Elasticsearch (E20)**
+
+Mattermost Enterprise E20 customers can enable `Elasticsearch <https://docs.mattermost.com/deployment/elasticsearch.html>`__ for optimized search performance at enterprise-scale. Elasticsearch solves many known issues with full text database search, such as dots, dashes, and email addresses returning unexpected results.
```
Comment (justinegeffen) [inline source/deployment/on-boarding.rst:135]:
```suggestion
Mattermost Enterprise Edition E20 customers can enable `Elasticsearch <https://docs.mattermost.com/deployment/elasticsearch.html>`__ for optimized search performance at enterprise-scale. Elasticsearch solves many known issues with full text database search, such as dots, dashes, and email addresses returning unexpected results.
```

File: source/deployment/on-boarding.rst [hunk 13]
```diff
@@ -67,3 +67,77 @@ Common Tasks
...
+
+**9. Enable Elasticsearch (E20)**
+
+Mattermost Enterprise E20 customers can enable `Elasticsearch <https://docs.mattermost.com/deployment/elasticsearch.html>`__ for optimized search performance at enterprise-scale. Elasticsearch solves many known issues with full text database search, such as dots, dashes, and email addresses returning unexpected results. 
+
+Enable Elasticsearch by configuring the following: **System Console > Elasticsearch > Enable Indexing**: ``true``. See the `Elasticsearch <https://docs.mattermost.com/administration/config-settings.html#elasticsearch>`__ configuration settings documentation for details. Enabling Elasticsearch requires `setting up an Elasticsearch server <https://docs.mattermost.com/deployment/elasticsearch.html#setting-up-an-elasticsearch-server>`__.
```
Comment (justinegeffen) [inline source/deployment/on-boarding.rst:137]:
```suggestion
Enable Elasticsearch by setting **System Console > Elasticsearch > Enable Indexing** to ``true``. See the `Elasticsearch <https://docs.mattermost.com/administration/config-settings.html#elasticsearch>`__ configuration settings documentation for details. Enabling Elasticsearch requires `setting up an Elasticsearch server <https://docs.mattermost.com/deployment/elasticsearch.html#setting-up-an-elasticsearch-server>`__.
```

File: source/deployment/on-boarding.rst [hunk 14]
```diff
@@ -67,3 +67,77 @@ Common Tasks
...
+
+**4. Enable Full Content Push Notifications**
+
+Enable push notifications on mobile devices to deliver messages in real time by configuring the following: **System Console > Push Notification Server > Enable Push Notifications**: ``Use TPNS``. See the `Push notification server <https://docs.mattermost.com/administration/config-settings.html#push-notification-server>`__ configuration settings documentation for details. 
+
+Enable full content push notifications, including the sender’s name, the channel name, and the message text, by configuring the following: **System Console > Notifications > Push Notification Contents**: ``Full message contents``. See the `Push notification contents <https://docs.mattermost.com/administration/config-settings.html#push-notification-contents>`__ configuration settings documentation for details.
```
Comment (justinegeffen) [inline source/deployment/on-boarding.rst:103]:
```suggestion
Enable full content push notifications, including the sender’s name, the channel name, and the message text, by setting **System Console > Notifications > Push Notification Contents** to ``Full message contents``. See the `Push notification contents <https://docs.mattermost.com/administration/config-settings.html#push-notification-contents>`__ configuration settings documentation for details.
```

File: source/deployment/on-boarding.rst [hunk 15]
```diff
@@ -67,3 +67,77 @@ Common Tasks
...
+
+**10. Try Experimental Features**
+
+Mattermost often releases experimental features to get user feedback on new functionality and product enhancements. For example, new `experimental sidebar features <https://mattermost.com/blog/dev-sneak-peek-experimental-sidebar-features>`__ include custom, collapsible channel categories, unread filtering, and many more enhancements for managing channels in your sidebar.  
+
+Enable experimental sidebar features by configuring the following: **System Console > Experimental > Experimental Sidebar Features**. See the `Experimental sidebar features (Experimental) <https://docs.mattermost.com/administration/config-settings.html#experimental-sidebar-features-experimental>`__ configuration settings documentation for details.
```
Comment (justinegeffen) [inline source/deployment/on-boarding.rst:143]:
```suggestion
Enable experimental sidebar features by enabling **System Console > Experimental > Experimental Sidebar Features**. See the `Experimental sidebar features (Experimental) <https://docs.mattermost.com/administration/config-settings.html#experimental-sidebar-features-experimental>`__ configuration settings documentation for details.
```

File: source/deployment/on-boarding.rst (hunk 2 above)
Comment (cwarnermm) [reply source/deployment/on-boarding.rst]: Sounds good, thanks!

File: source/deployment/on-boarding.rst (hunk 3 above)
Comment (cwarnermm) [reply source/deployment/on-boarding.rst]: Thanks!

File: source/deployment/on-boarding.rst (hunk 1 above)
Comment (cwarnermm) [reply source/deployment/on-boarding.rst:67]: Very good point. Thanks!

**Review by justinegeffen — APPROVED** [review]

**Comment by justinegeffen** [comment]
This is awesome, @cwarnermm! Such an excellent addition to our docs. 💯

**Review by esethna — APPROVED** [review]
Thanks Carrie!

**Pull request merged**